invertedai/api/blame.py

```diff
@@ -26,63 +26,64 @@ class BlameResponse(BaseModel):
+    reasons: Optional[Dict[int, List[str]]] #: A dictionary with agent IDs as keys and a list of fault class strings for why the keyed agent is to blame.
+    confidence_score: Optional[float] #: Float value between [0,1] indicating the models confidence in the response.
     birdviews: Optional[List[Image]]  #: If `get_birdviews` was set, this contains the resulting image.
 
 
 @validate_arguments
 def blame(
     location: str,
-    candidate_agents: Tuple[int, int],
+    colliding_agents: Tuple[int, int],
     agent_state_history: List[List[AgentState]],
     agent_attributes: List[AgentAttributes],
     traffic_light_state_history: Optional[List[TrafficLightStatesDict]] = None,
     get_reasons: bool = False,
     get_confidence_score: bool = False,
-    get_birdviews: bool = False,
-    detect_collisions: bool = False
+    get_birdviews: bool = False
 ) -> BlameResponse:
     """
     Parameters
     ----------
     location:
         Location name in IAI format.
 
-    candidate_agents:
-        Two agents involved in the collision.
+    colliding_agents:
+        Two agents involved in the collision. These integers should correspond to the 
+        indices of the relevant agents in the lists within agent_state_history.
 
     agent_state_history:
-        History of the states of all agents (up to 100 agents) right before the collision.
-        Each AgentStates is a list [x, y, orientation, speed] where x: [float], and
-        y: [float] are coordinate in meters, orientation: [float] is in radians with 0 pointing along x and pi/2
-        pointing along y and speed: [float] is in m/s. If an empty list or None is passed, 'num_agents_to_spawn'
-        unconditional agents will be spawned. If the number of agents passed is less than 'num_agents_to_spawn',
-        the remaining agents will be spawned conditionally on the provided agents. If 'num_agents_to_spawn' is equal
-        to the number of agents in `agent_state_history`, the simulation will be initialized only with the agents
-        provided. 'num_agents_to_spawn' cannot be less than the number of agents in `agent_state_history`.
+        Lists containing AgentState objects for every agent within the scene (up to 100
+        agents) for each time step within the relevant sequence immediately preceding 
+        the collision.
+        The final list of AgentState objects should include the first time step of the 
+        collision and no time steps afterwards. The lists of AgentState objects preceding 
+        the collision should capture enough of the scenario context before the collision 
+        for the Blame model to analyze and assign fault. 30-100 time steps preceding the 
+        the collision is recommended for best results.
+        Each AgentState state must include x: [float], y: [float] corrdinate in meters 
+        orientation: [float] in radians with 0 pointing along x and pi/2 pointing along 
```

Comment:
I would add  it's  positive x-axis and positive y-axis

Reply:
Done.